pkg/testutil/setup.go

```diff
@@ -9,15 +9,15 @@ import (
 	"strings"
 	"time"
 
+	"github.com/treeverse/lakefs/pkg/block"
```

Comment:
Don't use a separate block (I know it's the IDE default behaviour..)

Reply:
I'm using the goimports plugin which is supposed to fix it but it somehow doesn't.
Fixed - thanks